sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2222,15 +2222,15 @@ def to_dict(self) -> Dict:
         # CurrencyValue objects are interpreted as dict, therefore need to be processed first
         # to call the proper to_dict() method.
         if self.value_type == "currency":
-            value = self.value.to_dict()  # type: ignore
+            value = self.value.to_dict() if self.value else None  # type: ignore
         # AddressValue objects are interpreted as dict, therefore need to be processed first
         # to call the proper to_dict() method.
         elif self.value_type == "address":
-            value = self.value.to_dict()  # type: ignore
+            value = self.value.to_dict() if self.value else None  # type: ignore
```

Comment:
I'm wondering if currency and address should return {}

```suggestion
            value = self.value.to_dict() if self.value else {} # type: ignore
```

I'm thinking otherwise it might break the from_dict on these models.

Reply:
We've been doing None for types that are other models in general...but you're right I'll add something in the from dict logic to account for a None value...working on it!

Reply:
Done!